libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCredentials.m

```diff
@@ -140,15 +143,12 @@ - (void)encodeWithCoder:(NSCoder *)coder {
     [coder encodeObject:self.issuedAt           forKey:@"SFOAuthIssuedAt"];
     [coder encodeObject:self.protocol           forKey:@"SFOAuthProtocol"];
     [coder encodeObject:self.lightningDomain    forKey:@"SFOAuthLightningDomain"];
-    [coder encodeObject:self.lightningSid       forKey:@"SFOAuthLightningSID"];
```

Comment:
just curious, why do these encode calls go away?

Reply:
The sensitive stuff goes to the keychain and is handled by SFOAuthKeychainCredentials.
I stopped decoding them from the archive in the [last PR](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3771/files) but forgot to take out the encoding part.